File: src/dlls/CSSPWebModels/src/WebProvince.cs
﻿/*
 * Manually edited
 * 
 */
using CSSPDBModels;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSSPWebModels
{
    [NotMapped]
    public partial class WebProvince : WebBase
    {
        #region Properties
        public List<WebBase> TVItemParentList { get; set; }
        public List<WebBase> TVItemAreaList { get; set; }
        public List<SamplingPlan> SamplingPlanList { get; set; }
        #endregion Properties

        #region Constructors
        public WebProvince()
        {
            TVItemParentList = new List<WebBase>();
            TVItemAreaList = new List<WebBase>();
            SamplingPlanList = new List<SamplingPlan>();
        }
        #endregion Constructors
    }
}

File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/webprovince.ts
import { SamplingPlan } from './generated/SamplingPlan.model';
import { WebBase } from './webbase';

export class WebCountry extends WebBase {
    TVItemParentList: WebBase[] = [];
    TVItemAreaList: WebBase[] = [];
    SamplingPlanList: SamplingPlan[] = [];
}
